### PR TITLE
Authentication - finding a test URL

### PR DESCRIPTION
**Pull request by psiinon**
I think the more explicit we can make these instructions the better;)
A test URL will also be needed for the verification...

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Could we go with "indicator" URL or something like that?

Test URL seems like something you target, "indicator" also aligns more with what is seen in the GUI, etc.

File: site/content/docs/authentication/session-handling.md [hunk 1]
```diff
@@ -26,16 +26,42 @@ Session Management is difficult to test in isolation so you will typically need
 
 Server-side Session Management is nearly always handled by HTTP headers and is typically straightforward to configure - getting hold of the authentication tokens to put in the headers may well be harder.
 
+If you do not know what type of session management your application uses then authenticate manually to your app while proxying through ZAP and look at the responses from the app.
+
+#### Cookie-based Session Management
+
+If you can see any `Set-Cookie` (or possibly `SetCookie2`) headers being returned then it is likely that your app is using cookie-based session management.
```
Comment (kingthorin) [inline site/content/docs/authentication/session-handling.md:33]:
```suggestion
If you can see any `Set-Cookie` (or possibly `Set-Cookie2`) headers being returned then it is likely that your app is using cookie-based session management.
```

**Comment by psiinon** [comment]
"Verification"?
That would tie in with the terminology we use on the Session Properties / Authentication screen: "URL to Poll for Verification"

**Comment by kingthorin** [comment]
Yup "Verification" sounds good.

**Comment by psiinon** [comment]
Thanks - updated based on the feedback

**Comment by psiinon** [comment]
Removed the `Set-Cookie2` bit as it looks like no browsers support it any more...

File: site/content/docs/authentication/finding-a-verification-url.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- authentication
+- guide
+type: page
+---
+
+If you need to set up ZAP to handle authentication then you really need to find a suitable verification URL in your app.
+
+The verification URL will be one that you can request from the ZAP Manual Request Editor and which will send a response that allows you to work out whether you are logged in or not. What that looks like will entirely depend on your app, but could be something like:
+
+* Any page or API endpoint which returns the authenticated  user's Id
```
Comment (kingthorin) [inline site/content/docs/authentication/finding-a-verification-url.md:13]: There's still a double space here before "user's"

Comment (psiinon) [reply site/content/docs/authentication/finding-a-verification-url.md]: Not any more - fixed in the last push ;)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**